diff --git a/particula/next/dynamics/coagulation/transition_regime.py b/particula/next/dynamics/coagulation/transition_regime.py
--- particula/next/dynamics/coagulation/transition_regime.py
+++ particula/next/dynamics/coagulation/transition_regime.py
@@ -73,9 +73,9 @@
     Journal of Chemical Physics, 126(12).
     https://doi.org/10.1063/1.2713719
     """
     coulomb_potential_ratio = np.maximum(
-        coulomb_potential_ratio, 1e-16
+        coulomb_potential_ratio, 1e-14
     )  # Avoid division by zero
 
     continuum_limit = 4 * np.pi * diffusive_knudsen**2
     kinetic = coulomb_enhancement.kinetic(coulomb_potential_ratio)
