diff --git a/pyproject.toml b/pyproject.toml
--- pyproject.toml
+++ pyproject.toml
@@ -22,9 +22,9 @@
 ]
 
 dependencies = [
     "numpy", "scipy", "pint", "hypersolver==0.0.8", "pandas",
-    "netCDF4", "shapely", "PyMieScatt", "tqdm", "pytz", "typing",
+    "netCDF4", "shapely", "PyMieScatt", "tqdm", "pytz",
     "matplotlib"
 ]
 
 # This is set automatically by flit using `particula.__version__`
